docs/cfn-service-role.yml

```diff
@@ -135,42 +185,96 @@ Resources:
                   - lambda:DeleteLayerVersion
                   - lambda:GetEventSourceMapping
                   - lambda:GetFunction
+                  - lambda:GetFunctionCodeSigningConfig
+                  - lambda:GetFunctionConfiguration
                   - lambda:GetLayerVersion
+                  - lambda:GetRuntimeManagementConfig
                   - lambda:InvokeFunction
                   - lambda:PublishLayerVersion
+                  - lambda:PublishVersion
                   - lambda:PutFunctionConcurrency
                   - lambda:RemovePermission
                   - lambda:UpdateFunctionCode
                   - lambda:UpdateFunctionConfiguration
                   - logs:CreateLogGroup
+                  - logs:CreateLogStream
                   - logs:DeleteLogGroup
+                  - logs:DeleteLogStream
                   - logs:DeleteResourcePolicy
                   - logs:DescribeLogGroups
+                  - logs:DescribeLogStreams
                   - logs:DescribeResourcePolicies
                   - logs:PutResourcePolicy
                   - logs:PutRetentionPolicy
+                  - organizations:ListAWSServiceAccessForOrganization
                   - rds:AddTagsToResource
                   - rds:CreateDBInstance
                   - rds:CreateDBSubnetGroup
+                  - rds:CreateTenantDatabase
                   - rds:DeleteDBInstance
                   - rds:DeleteDBSubnetGroup
+                  - rds:DeleteTenantDatabase
                   - rds:DescribeDBInstances
                   - rds:DescribeDBSubnetGroups
+                  - rds:ListTagsForResource
                   - rds:ModifyDBInstance
                   - s3:CreateBucket
                   - s3:DeleteBucket
                   - s3:DeleteBucketPolicy
+                  - s3:GetAccelerateConfiguration
+                  - s3:GetAnalyticsConfiguration
+                  - s3:GetBucketCORS
+                  - s3:GetBucketLocation
+                  - s3:GetBucketLogging
+                  - s3:GetBucketNotification
+                  - s3:GetBucketObjectLockConfiguration
+                  - s3:GetBucketOwnershipControls
                   - s3:GetBucketPolicy
+                  - s3:GetBucketPublicAccessBlock
+                  - s3:GetBucketTagging
+                  - s3:GetBucketVersioning
+                  - s3:GetBucketWebsite
+                  - s3:GetEncryptionConfiguration
+                  - s3:GetIntelligentTieringConfiguration
+                  - s3:GetInventoryConfiguration
+                  - s3:GetLifecycleConfiguration
+                  - s3:GetMetricsConfiguration
                   - s3:GetObject
+                  - s3:GetReplicationConfiguration
```

Comment:
Is this a currently used IAM policy? Or potentially future state?

Reply:
We take * now (as the stack has). Believe it or not this is likely a current operation as reported by CloudTrail and Generate Policy.